pkg/acsengine/k8s_versions.go

```diff
@@ -8,6 +8,34 @@ import (
 
 // KubeConfigs represents Docker images used for Kubernetes components based on Kubernetes versions (major.minor.patch)
 var KubeConfigs = map[string]map[string]string{
+	common.KubernetesVersion1Dot9Dot2: {
+		"hyperkube": "hyperkube-amd64:v1.9.2",
+		"ccm":       "cloud-controller-manager-amd64:v1.9.2",
+		"dockerEngineVersion":       "1.12.*",
+		DefaultDashboardAddonName:   "kubernetes-dashboard-amd64:v1.8.2",
+		"exechealthz":               "exechealthz-amd64:1.2",
+		"addonresizer":              "addon-resizer:1.8.1",
```

Comment:
See https://github.com/Azure/acs-engine/pull/2090, we need to revert to `1.7` for addon-resizer.